examples/complete/main.tf

```diff
@@ -3,6 +3,7 @@ resource "aws_cloudwatch_log_group" "this" {
   count             = local.logging != "OVERRIDE" ? 0 : 1
   name              = "${local.name}-log-group"
   retention_in_days = 7
+  kms_key_id        = module.kms_key.key_id
```

Comment:
I understand you are adding an external kms key, does this module supports `aws:sse` and `aws:kms`?

The rule of thumb should be:
Default: `aws:kms` which is the aws managed kms key in the account
Complete Example: cmk 

The module should also allow to create its own custom KMS key.

Will the kms encryption be applied to both the cloud watch but also the session to the containers?

Should we have the option to use a single KMS key or separate KMS keys?

If this features aren't done it would be good to add them to the changelog to address them in future releases.

Also review the CHANGELOG for ECS and see if there are any new features missing which where recently released so you can add them for future development.

Reply:
I have added an option to create a single kms key for the module in the chaneglog.